utils/nctl/sh/scenarios/gov96.sh

```diff
@@ -28,35 +28,35 @@ function main() {
     do_await_era_change
     # 2. Verify all nodes are in sync
     check_network_sync
-    # 3-5. Stop three nodes
+    # 3-6. Stop four nodes
     do_stop_node "5"
     do_stop_node "4"
     do_stop_node "3"
     do_stop_node "2"
-    # 6. Ensure chain stalled
+    # 7. Ensure chain stalled
     # Eras are less than a minute long. 
     # We want to make sure that consensus handles being resumed,
     # even after being "paused" for multiples of era length.
     assert_chain_stalled "300"
-    # 7. Verify that the running node can still handle queries.
+    # 8. Verify that the running node can still handle queries.
     assert_node_operational "1"
-    # 8-10. Restart three nodes
+    # 9-12. Restart four nodes
     do_start_node "5" "$STALLED_LFB"
     do_start_node "4" "$STALLED_LFB"
     do_start_node "3" "$STALLED_LFB"
     do_start_node "2" "$STALLED_LFB"
-    # 11. Verify all nodes are in sync
+    # 13. Verify all nodes are in sync
     check_network_sync
-    # 12. Ensure era proceeds after restart
+    # 14. Ensure era proceeds after restart
     do_await_era_change "2"
-    # 13. Verify all nodes are in sync
+    # 15. Verify all nodes are in sync
     check_network_sync
-    # 14-16. Compare stalled lfb hash to currentm
+    # 16-17. Compare stalled lfb hash to current
```

Comment:
```suggestion
    # 16-19. Compare stalled lfb hash to current
```
(Would be easier to just remove the numbers…)